spec/integration/rest/api/v2010/account/incomingPhoneNumber.spec.js

```diff
@@ -426,6 +426,132 @@ describe('IncomingPhoneNumber', function() {
                       .incomingPhoneNumbers.each({callback: () => done()}, () => fail('wrong callback!'));
     }
   );
+  it('should call done in the opts object when done',
```

Comment:
These new tests will get overwritten during the next release since this is a generated file. Should move them to a file under `spec/unit` which contains hand-crafted tests.

Reply:
Thanks for the input. I'll go ahead and make changes to move my hand-crafted tests over into the `spec/unit` space.

Reply:
The hand-crafted unit tests have now been moved to the `spec/unit` space.